ios/FluentUI/People Picker/PersonaListView.swift

```diff
@@ -194,14 +194,12 @@ extension PersonaListView: UITableViewDataSource {
                 guard let cell = dequeueReusableCell(withIdentifier: ActivityIndicatorCell.identifier, for: indexPath) as? ActivityIndicatorCell else {
                     return UITableViewCell()
                 }
-                cell.hideSystemSeparator()
```

Comment:
Do you not need to replace this with:

// hide system separator so we can draw our own...
separatorInset = UIEdgeInsets(top: 0, left: CGFloat.greatestFiniteMagnitude, bottom: 0, right: 0)

Like you do in ActionsCell?

Reply:
And same with the other two instances

Reply:
this is on purpose.  because the personlistview the tableview itself has hideseparator style.